docs/src/terminology.md

```diff
@@ -22,23 +32,27 @@ A contiguous set of [entries](terminology.md#entry) on the ledger covered by a [
 
 ## blockhash
 
-A preimage resistant [hash](terminology.md#hash) of the [ledger](terminology.md#ledger) at a given [block height](terminology.md#block-height). Taken from the last [entry id](terminology.md#entry-id) in the slot
+A fingerprint ([hash](terminology.md#hash)) that identifies a record (block).  Solana computes a blockhash from the last [entry id](terminology.md#entry-id) of the block.
 
 ## block height
 
 The number of [blocks](terminology.md#block) beneath the current block. The first block after the [genesis block](terminology.md#genesis-block) has height one.
 
 ## bootstrap validator
 
-The first [validator](terminology.md#validator) to produce a [block](terminology.md#block).
+The server node ([validator](terminology.md#validator)) that produces the genesis (first) [block](terminology.md#block) of a block chain.
+
+## BPF loader 
+
+The Solana program that owns and loads smart contract programs into validator memory.
```

Comment:
This is not quite accurate, should be:
`The Solana program that owns and loads BPF smart contract programs, allowing the program to interface with the runtime.`